blazor/visual-studio-integration/upgrade-project.md

```diff
@@ -11,17 +11,17 @@ documentation: ug
 
 The Syncfusion Blazor migration add-in for Visual Studio allows you to migrate an existing Syncfusion Blazor application from one version of Essential Studio version to another version. This reduces the amount of manual work required when migrating the Syncfusion version.
 
-The steps below will assist you to upgrade the Syncfusion version in the Syncfusion Blazor application via Visual Studio 2019 or Visual Studio 2022:
+The steps below will assist you to upgrade the Syncfusion version in the Syncfusion Blazor application via Visual Visual Studio 2022:

```

Comment:
remove repeated words

Reply:
Removed repeated words in existing content.